x/protocolpool/types/params.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"errors"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func DefaultParams() Params {
+	return Params{
+		EnabledDistributionDenoms: []string{sdk.DefaultBondDenom},
+		DistributionFrequency:     1,
+	}
+}
+
+func (p *Params) Validate() error {
+	seenDenoms := make(map[string]struct{})
+
+	for _, d := range p.EnabledDistributionDenoms {
+		err := sdk.ValidateDenom(d)
+		if err != nil {
+			return err
+		}
+
+		if _, seen := seenDenoms[d]; seen {
+			return fmt.Errorf("duplicate enabled distribution denom %s", d)
+		}
+		seenDenoms[d] = struct{}{}
+	}
+
+	if p.DistributionFrequency == 0 {
+		return errors.New("DistributionFrequency must be greater than 0")
+	}
```

Comment:
what if someone wanted to use protocolpool as a static treasury that doesn't ever distribute on a regular basis? or would we jsut advise them _not_ to use protocolpool here?

Reply:
I'm not sure what you mean here. You need `DistributionFrequency` to be nonzero so that the funds from `x/distribution` will actually fill the protocolpool module account.  Otherwise, they will just remain in the escrow account forever

Reply:
got it, forgot that we distribute to module account at the end of the begin blocker